inc/admin/admin.php

```diff
@@ -131,6 +131,14 @@ function alcatraz_page_options_metabox() {
 		),
 	) );
 
+	// Hide Title.
+	$page_options->add_field( array(
+		'name' => __( 'Hide Page Title?', 'alcatraz' ),
+		'desc' => __( '', 'alcatraz' ),
```

Comment:
Actually, the whole `'desc' => '',` line doesn't need to be there at all, since it is empty.